default/service-settings/deck.yml

```diff
@@ -1,3 +1,3 @@
-artifactId: quay.io/opsmxpublic/ubi8-oes-deck:3.5.1
+artifactId: quay.io/opsmxpublic/ubi8-oes-deck:3.7.2
```

Comment:
I deck was NOT to be updated as there is a bug in the 1.26.6 . Can you please check with Anoop T and decide?